src/cljdoc/render.clj

```diff
@@ -40,7 +40,13 @@
                    first)
         [doc-type contents] (doctree/entry->type-and-content doc-p)
         top-bar-component (layout/top-bar version-entity (-> cache-bundle :version :scm :url))
-        sidebar-contents (sidebar/sidebar-contents route-params cache-bundle last-build)]
+        sidebar-contents (sidebar/sidebar-contents route-params cache-bundle last-build)
+        articles-block (doctree/get-neighbour-entries (remove #(contains? #{"Readme" "Changelog"}
```

Comment:
Thanks for the PR @moonbrv! 😃 

Should `remove` happen after flattening the `doc-tree`?  or "Readme" and "Changelog" are always expected in the root so it doesn't matter?

Reply:
I decide to remove `Readme` and `Changelog` to have the same data as in the sidebar. I think for the navigating we need only articles (`doc-tree-with-rest` in the code snippet below`). I can change if need 

https://github.com/cljdoc/cljdoc/blob/d6c6fab0ef34dbb2bf3734ba0614466a401c20bb/src/cljdoc/render/sidebar.clj#L46-L50